lec_7_HW/task_2.js

```diff
@@ -0,0 +1,74 @@
+const str = "I am the best AQA ever!";
+
+function replaceLetterToCount(str) {
+    let count = {};
+
+    str = str.toLowerCase();
+
+    for (let char of str) {
+        if (char.match(/[a-z]/i)) {
+            count[char] = (count[char] || 0) + 1;
```

Comment:
As option, because you do extra step 
1) assign zero
2) increase by one
-
1) assign one or increase
```suggestion
            count[char] ? count[char] + 1 : count[char] = 1;
```

Reply:
```suggestion
            count[char] = count[char] + 1 :  1;
```